sound/soc/sof/ipc4-topology.h

```diff
@@ -359,4 +359,39 @@ struct sof_ipc4_process {
 	bool payload_with_output_fmt;
 };
 
+/**
```

Comment:
Commit message:
"This structure is mainly used to interpret byte kcontrol data" -> "This structure is mainly used to interpret the bytes data of the module", we don't have kcontrol support yet for bytes type, we only store the blob from the topology under cdata of the control.

Also to note that this is only done for blob type of MOD_INIT_INSTANCE, other type of blobs are not inspected.